postcss.config.js

```diff
@@ -1,15 +1,14 @@
+const plugins = [
+  require('postcss-inline-svg')({
+    paths: ['./src/assets'],
+  }),
+];
+
+// For production only, add cssnano
+if (/prod/.test(process.env.NODE_ENV)) {
+  plugins.push(require('cssnano')());
+}
```

Comment:
Is this change necessary for this PR?

Reply:
None of the changes having to do with postcss configuration should be necessary for this PR. They were introduced based on initial feedback from @calebeby, before I decided to completely remove `postcss-preset-env`. I can revert all of them.

When I discussed with @calebeby, the feeling was that the gulp task should take plugins. I wasn't clear that this happened automatically (referring to https://github.com/cloudfour/cloudfour.com-patterns/pull/662#discussion_r416905806)